### ISSUE TITLE
Fade doesn't work in notification controls, PiP controls and headset buttons

### ISSUE THREAD

**Issue by furrydaemon** [fast4x/RiMusic#4710]
### Version of RiMusic

0.6.61

### Android version

13

### I've done these:

- [X] Uninstall old build before installing a new one
- [X] Checked opening issues about my similar problem

### Tell us how you trigger this bug

1. Open RiMusic
2. Play any song and use the notification buttons, headset buttons or PiP player buttons
3. Fade may work, may have a hike in the volume and then fade to regain volume or it may just not work at all

### Submit a video

https://github.com/user-attachments/assets/5f484656-ea87-4763-9e15-03f9049b8f20

### What was your expectation?

Fade audio duh, what else, the option is fade and it doesnt fade lmao

### What more can you tell us? (Optional)

Also add #4648 :v thx for the great support :3

**Comment by fast4x** [comment]
Yep you are right, isn't a bug, for now not possible to re implement...

**Comment by Ghiato** [comment]
Yes, a crossfade for song transitions wouldn't be bad.

**Comment by furrydaemon** [comment]
> Yep you are right, isn't a bug, for now not possible to re implement...

I don't understand, what is the problem? The most notorious issue is that pressing the pause button cuts the music inmediatly instead of fading it away

**Comment by fast4x** [comment]
It’s not a problem, the player has been completely rewritten and in a few days you will be able to have this function and others. You must have patience!
